axonius_api_client/http.py

```diff
@@ -171,6 +172,12 @@ class Http:
     """Client object that created this object."""
     # TBD: Connect needs an interface for proper type hinting without circular reference
 
+    RETRY_COUNT: t.Optional[int] = 3
+    """Number of times to retry a request if it fails."""
+
+    RETRY_BACKOFF: t.Optional[int] = 1
+    """Number of seconds to wait between retries. This value will be doubled after each retry."""
```

Comment:
@nate-axonius it wont be doubled, its multipled by the number of retries right? 
make the default to 5 sec